ConsoleApp/Program.cs

```diff
@@ -0,0 +1,96 @@
+﻿// homework/06-arrays
+
+
+
+
+// bubble sort
+/*
+Works by repeatedly swapping the adjacent elements if they are in the wrong order. 
+This algorithm is not suitable for large data sets as its average and worst-case time complexity is quite high.
+*/
+static int[] BubbleSort(int[] arr)
+{
+    int length = arr.Length;
+    bool flag = new bool(); //прапорець по замовчуванню false
+    do

```

Comment:
do/while is not needed, yo uwill always finish with flag === false
as well, better check `!flag`